tensorboard/plugins/profile/pod_viewer/stack_bar_chart/stack-bar-chart.ts

```diff
@@ -0,0 +1,203 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+namespace pod_viewer_stack_bar_chart {
+
+Polymer({
+  is: 'stack-bar-chart',
+  properties: {
+    id: {
+      type: String,
+    },
+    data: {
+      type: Object,
+      notify: true,
```

Comment:
AFAICT, data is not being set in this module, thus, should not be notified.

Reply:
Done.